api/client/node/update.go

```diff
@@ -27,6 +28,8 @@ func newUpdateCommand(dockerCli *client.DockerCli) *cobra.Command {
 	flags.StringVar(&opts.role, flagRole, "", "Role of the node (worker/manager)")
 	flags.StringVar(&opts.membership, flagMembership, "", "Membership of the node (accepted/rejected)")
 	flags.StringVar(&opts.availability, flagAvailability, "", "Availability of the node (active/pause/drain)")
+	flags.StringSlice(flagLabelAdd, []string{}, "Add or update node labels (key1=value1,key2=value2)")
+	flags.StringSlice(flagLabelRemove, []string{}, "Remove node labels (key1,key2)")
```

Comment:
Noticed this while reviewing https://github.com/docker/docker/pull/24507#discussion_r70327976

Looks like we use a `StringSlice` here, which makes this flag support a comma-separated list. https://github.com/docker/docker/blob/28676fc04b9a307718b66bd28e8d9281d05f38ac/vendor/src/github.com/spf13/pflag/string_slice.go#L46-L54

We don't support this on other flags (and not on `--label`), so I don't think we should support that here